 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
10.20.0.53 - - [04/May/2016 15:21:14] "GET /db/api/thread/listPosts/?limit=74&order=desc&thread=7543 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:14] "GET /db/api/user/listFollowers/?limit=91&user=r46ul%40gmail.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:14] "GET /db/api/user/details/?user=cqdf%40ya.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:15] "GET /db/api/thread/list/?forum=se4iu&limit=82&order=asc&thread=2064 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:16] "GET /db/api/forum/listPosts/?related=user&limit=47&order=asc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:16] "GET /db/api/forum/listThreads/?related=user&limit=73&order=asc&forum=dltm4sp HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:21:16] "GET /db/api/user/listFollowing/?limit=29&user=flxrh8di7%40ua.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:21:16] "GET /db/api/user/listFollowers/?limit=42&user=yt2%40yahoo.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:17] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=97&order=desc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:17] "GET /db/api/user/listFollowers/?limit=61&user=nx%40ya.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:25] "GET /db/api/forum/listUsers/?related=forum&limit=76&order=desc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:25] "GET /db/api/thread/listPosts/?limit=63&order=asc&thread=7103 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:33] "GET /db/api/forum/listUsers/?related=forum&related=user&limit=49&order=desc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:33] "GET /db/api/forum/details/?forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:34] "GET /db/api/forum/listPosts/?related=thread&limit=35&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:34] "GET /db/api/thread/details/?thread=1505 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:34] "GET /db/api/thread/listPosts/?limit=51&order=desc&thread=8250 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:34] "GET /db/api/forum/details/?related=user&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:34] "GET /db/api/post/details/?post=476616&related=user&related=thread HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:34] "GET /db/api/thread/details/?related=user&related=forum&thread=5377 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:34] "GET /db/api/forum/details/?related=user&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:34] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=82&order=asc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:42] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=72&order=asc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:42] "GET /db/api/user/listFollowers/?limit=17&user=a0c2eif%40ya.ru&order=desc HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:21:42] "GET /db/api/user/listFollowing/?limit=98&user=hvkr1g0d%40bk.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:21:42] "GET /db/api/thread/list/?forum=cmq7igz2&limit=93&order=asc&thread=3472 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/forum/listUsers/?related=forum&limit=59&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/post/list/?post=325538&limit=22&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/thread/listPosts/?limit=47&order=desc&thread=5220 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/forum/listThreads/?limit=46&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/user/listFollowers/?limit=83&user=r0emw73%40gmail.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/forum/listThreads/?related=user&limit=49&order=desc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/forum/details/?related=user&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/thread/listPosts/?limit=26&order=asc&thread=6485 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/forum/listPosts/?limit=79&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/post/list/?post=466951&limit=38&order=asc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:49] "GET /db/api/forum/listThreads/?related=forum&limit=14&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:21:57] "GET /db/api/forum/listUsers/?limit=65&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:21:57] "GET /db/api/user/listFollowing/?limit=41&user=u2xkov%40gmail.com&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:21:57] "GET /db/api/thread/details/?thread=3986 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:07] "GET /db/api/forum/listUsers/?related=forum&related=user&limit=52&order=asc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:08] "GET /db/api/forum/listThreads/?related=user&limit=83&order=desc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:18] "GET /db/api/forum/listUsers/?limit=51&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:29] "GET /db/api/forum/listUsers/?related=user&limit=30&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:29] "GET /db/api/thread/list/?forum=g91u&limit=69&order=desc&thread=5458 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:29] "GET /db/api/forum/details/?forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:29] "GET /db/api/user/listFollowers/?limit=74&user=2fesarnw%40gmail.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:30] "GET /db/api/forum/listPosts/?related=thread&related=forum&related=user&limit=71&order=desc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:31] "GET /db/api/forum/listThreads/?limit=85&order=desc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:31] "GET /db/api/forum/listThreads/?limit=53&order=asc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:32] "GET /db/api/thread/list/?forum=dltm4sp&limit=55&order=asc&thread=9087 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:32] "GET /db/api/user/details/?user=4bvrwm%40bk.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:41] "GET /db/api/forum/listUsers/?related=forum&related=user&limit=12&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:41] "GET /db/api/post/details/?post=537149&related=forum&related=thread&related=user HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:41] "GET /db/api/forum/details/?forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:41] "GET /db/api/post/list/?post=100967&limit=69&order=desc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:41] "GET /db/api/thread/details/?related=user&related=forum&thread=7239 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:41] "GET /db/api/user/listFollowers/?limit=100&user=zfmqu%40yahoo.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:41] "GET /db/api/forum/details/?forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:42] "GET /db/api/thread/list/?forum=yfo2qgghp6&limit=54&order=asc&thread=8733 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:42] "GET /db/api/post/list/?post=172835&limit=53&order=desc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:42] "GET /db/api/forum/details/?forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:42] "GET /db/api/post/details/?post=861713 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:42] "GET /db/api/forum/listThreads/?related=forum&limit=89&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:42] "GET /db/api/post/details/?post=659599&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:42] "GET /db/api/forum/details/?related=user&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:43] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=70&order=asc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:43] "GET /db/api/thread/listPosts/?limit=53&order=asc&thread=9170 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:43] "GET /db/api/post/details/?post=521963&related=user HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:43] "GET /db/api/forum/listPosts/?limit=84&order=asc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:53] "GET /db/api/forum/listUsers/?related=user&limit=63&order=asc&forum=se4iu HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:22:53] "GET /db/api/user/listFollowing/?limit=33&user=skh%40mail.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:22:53] "GET /db/api/forum/listPosts/?limit=23&order=asc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:53] "GET /db/api/thread/listPosts/?limit=80&order=asc&thread=407 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:53] "GET /db/api/forum/details/?forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:53] "GET /db/api/thread/listPosts/?limit=96&order=desc&thread=9231 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:53] "GET /db/api/user/listFollowers/?limit=81&user=48c%40list.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:54] "GET /db/api/post/list/?post=152716&limit=74&order=asc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:54] "GET /db/api/thread/list/?limit=28&order=desc&thread=3880&user=6v7nr%40mail.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:54] "GET /db/api/post/list/?post=772544&limit=54&order=asc&thread=7283 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:54] "GET /db/api/post/details/?post=856917&related=user&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:54] "GET /db/api/post/list/?post=933370&limit=86&order=desc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:54] "GET /db/api/forum/details/?forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:54] "GET /db/api/forum/details/?related=user&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:56] "GET /db/api/forum/listPosts/?related=thread&related=forum&limit=99&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:56] "GET /db/api/post/list/?post=706742&limit=90&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:56] "GET /db/api/user/listFollowers/?limit=62&user=5btx6s%40bk.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:22:56] "GET /db/api/user/details/?user=1rnuovzph3%40list.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:06] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=97&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:07] "GET /db/api/forum/listThreads/?limit=99&order=desc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:07] "GET /db/api/user/details/?user=nams%40gmail.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:07] "GET /db/api/post/list/?post=119039&limit=100&order=asc&thread=8628 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:07] "GET /db/api/user/listPosts/?limit=86&user=hu4r3fd18k%40bk.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:07] "GET /db/api/user/details/?user=b3zroau85h%40ya.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:17] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=78&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:17] "GET /db/api/thread/listPosts/?limit=93&order=desc&thread=3690 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:17] "GET /db/api/post/list/?post=711986&limit=59&order=desc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:17] "GET /db/api/thread/details/?related=forum&related=user&thread=6914 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:17] "GET /db/api/forum/details/?forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:17] "GET /db/api/user/details/?user=3tgwn6epv1%40ua.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:17] "GET /db/api/user/listFollowers/?limit=62&user=o3xud%40ua.ru&order=desc HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:23:17] "GET /db/api/user/listFollowing/?limit=45&user=i7uv29qsft%40ua.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:23:17] "GET /db/api/post/details/?post=603184&related=user&related=thread HTTP/1.1" 200 -
Exception on /db/api/thread/vote/ [POST]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/Thread.py", line 334, in voteThread
    if "vote" in request.json and "thread" in request.json:
TypeError: argument of type 'NoneType' is not iterable
10.20.0.53 - - [04/May/2016 15:23:17] "POST /db/api/thread/vote/ HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:23:18] "GET /db/api/forum/listThreads/?related=forum&limit=52&order=desc&forum=ql HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:23:18] "GET /db/api/user/listFollowing/?limit=34&user=4mtgy6hg%40ua.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:23:18] "GET /db/api/thread/list/?limit=34&order=desc&thread=1673&user=xyaonf2ud%40bk.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:18] "GET /db/api/forum/details/?forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:18] "GET /db/api/thread/details/?related=forum&thread=1387 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:19] "GET /db/api/forum/listPosts/?related=user&related=forum&limit=22&order=asc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:19] "POST /db/api/thread/update/ HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:19] "GET /db/api/forum/details/?related=user&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:20] "GET /db/api/thread/list/?forum=tomer&limit=39&order=asc&thread=6277 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:20] "GET /db/api/forum/details/?forum=yfo2qgghp6 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:23:20] "GET /db/api/user/listFollowing/?limit=33&user=z9%40gmail.com&order=desc HTTP/1.1" 500 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:23:20] "GET /db/api/user/listFollowing/?limit=100&user=bgztp65sik%40gmail.com&order=asc HTTP/1.1" 500 -
Exception on /db/api/post/restore/ [POST]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/Post.py", line 148, in restorePost
    if "post" in request.json:
TypeError: argument of type 'NoneType' is not iterable
10.20.0.53 - - [04/May/2016 15:23:20] "POST /db/api/post/restore/ HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:23:20] "GET /db/api/user/details/?user=aicx9%40yahoo.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:20] "GET /db/api/post/details/?post=179986&related=forum&related=thread HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:30] "GET /db/api/forum/listUsers/?related=forum&limit=15&order=asc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:30] "GET /db/api/forum/details/?related=user&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:30] "GET /db/api/user/details/?user=qz%40gmail.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:30] "GET /db/api/thread/details/?related=user&related=forum&thread=2459 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:30] "GET /db/api/user/listFollowers/?limit=32&user=pans482q%40bk.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:30] "GET /db/api/thread/listPosts/?limit=31&order=asc&thread=148 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:30] "GET /db/api/user/listPosts/?limit=26&user=3uatk5m6%40yahoo.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:30] "GET /db/api/user/listPosts/?limit=22&user=b8dyul%40gmail.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:30] "GET /db/api/forum/listThreads/?related=forum&limit=24&order=asc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:30] "GET /db/api/thread/details/?related=forum&thread=2064 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:31] "GET /db/api/forum/listThreads/?limit=72&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:31] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=60&order=desc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:42] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=62&order=desc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:42] "GET /db/api/forum/listPosts/?related=forum&limit=15&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:42] "GET /db/api/post/list/?post=918321&limit=25&order=asc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:52] "GET /db/api/forum/listUsers/?related=forum&limit=38&order=desc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:52] "GET /db/api/user/details/?user=vn8lut301i%40bk.ru HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:23:52] "GET /db/api/user/listFollowing/?limit=55&user=b7%40ya.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:23:52] "GET /db/api/user/listFollowers/?limit=100&user=zgq7%40ya.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:54] "GET /db/api/forum/listPosts/?related=thread&related=forum&related=user&limit=95&order=asc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:54] "GET /db/api/post/details/?post=523434&related=thread&related=user&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:54] "GET /db/api/user/listPosts/?limit=55&user=s3lrc%40gmail.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:54] "GET /db/api/user/listPosts/?limit=69&user=3k%40ua.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:54] "GET /db/api/forum/details/?forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:23:54] "GET /db/api/forum/listPosts/?related=user&limit=37&order=asc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:04] "GET /db/api/forum/listUsers/?related=user&limit=23&order=desc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:04] "GET /db/api/thread/listPosts/?limit=92&order=asc&thread=980 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:24:04] "GET /db/api/user/listFollowing/?limit=20&user=pu%40ya.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:24:04] "GET /db/api/user/listPosts/?limit=19&user=w5umgl80%40yahoo.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:04] "GET /db/api/user/listPosts/?limit=82&user=xi904nh8lv%40ua.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:04] "GET /db/api/user/details/?user=a4hmlg9y%40bk.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:04] "GET /db/api/post/list/?post=44223&limit=38&order=desc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:04] "GET /db/api/post/details/?post=21028&related=user&related=thread&related=forum HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:24:04] "GET /db/api/user/listFollowing/?limit=93&user=qu6%40mail.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:24:13] "GET /db/api/forum/listUsers/?related=forum&related=user&limit=38&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:13] "GET /db/api/post/details/?post=577735&related=thread&related=user HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:13] "GET /db/api/post/list/?post=964792&limit=40&order=asc&thread=5229 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:13] "GET /db/api/user/listFollowers/?limit=34&user=evx6z9lgna%40list.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:15] "GET /db/api/thread/list/?forum=cmxageot3&limit=95&order=asc&thread=6689 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:15] "GET /db/api/forum/listPosts/?limit=16&order=asc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:15] "GET /db/api/forum/listThreads/?related=forum&limit=37&order=asc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:15] "GET /db/api/user/listPosts/?limit=60&user=7mi5f6pget%40mail.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:15] "GET /db/api/thread/listPosts/?limit=55&order=desc&thread=3400 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:23] "GET /db/api/forum/listUsers/?limit=95&order=asc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:23] "GET /db/api/post/list/?post=215094&limit=15&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:23] "GET /db/api/user/listFollowers/?limit=38&user=5v8%40list.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:24] "GET /db/api/forum/listPosts/?related=forum&related=thread&limit=47&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:24] "GET /db/api/thread/listPosts/?limit=10&order=desc&thread=3043 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:24] "GET /db/api/forum/details/?related=user&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:24] "GET /db/api/forum/listPosts/?limit=16&order=desc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:24] "GET /db/api/forum/details/?forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:33] "GET /db/api/forum/listUsers/?limit=97&order=asc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:33] "GET /db/api/user/listFollowing/?limit=83&user=ucw%40mail.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:33] "GET /db/api/thread/listPosts/?limit=35&order=asc&thread=3656 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:40] "GET /db/api/forum/listUsers/?limit=12&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:40] "GET /db/api/post/details/?post=99513 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:40] "GET /db/api/post/list/?post=735672&limit=74&order=asc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:40] "GET /db/api/user/details/?user=xsga%40gmail.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:40] "GET /db/api/post/details/?post=534267&related=thread&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:47] "GET /db/api/forum/listUsers/?related=forum&related=user&limit=79&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:48] "GET /db/api/forum/listPosts/?related=forum&related=thread&related=user&limit=65&order=desc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:48] "GET /db/api/user/listFollowers/?limit=39&user=dyk6rqw%40yahoo.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:48] "GET /db/api/post/details/?post=341289 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:24:48] "GET /db/api/user/listFollowing/?limit=33&user=ph%40ua.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=51&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/user/listFollowers/?limit=42&user=rwv13bqeas%40bk.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/forum/listPosts/?related=forum&related=thread&limit=41&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/thread/list/?forum=cmxageot3&limit=48&order=desc&thread=791 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/forum/listPosts/?related=user&related=forum&limit=40&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "POST /db/api/thread/create/ HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/thread/listPosts/?limit=15&order=desc&thread=7554 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/user/details/?user=7v8bnk29xw%40gmail.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/forum/details/?related=user&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/thread/details/?thread=7729 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/user/details/?user=lxegpo781%40bk.ru HTTP/1.1" 200 -
Exception on /db/api/thread/vote/ [POST]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/Thread.py", line 334, in voteThread
    if "vote" in request.json and "thread" in request.json:
TypeError: argument of type 'NoneType' is not iterable
10.20.0.53 - - [04/May/2016 15:24:49] "POST /db/api/thread/vote/ HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/user/details/?user=kmvuinb2dp%40yahoo.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:49] "GET /db/api/user/listPosts/?limit=35&user=axg8ns7%40yahoo.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:50] "GET /db/api/thread/listPosts/?limit=58&order=asc&thread=7012 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:50] "GET /db/api/user/listPosts/?limit=47&user=qgdb%40yahoo.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:50] "GET /db/api/thread/listPosts/?limit=26&order=asc&thread=2333 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:50] "GET /db/api/user/listFollowers/?limit=60&user=t452%40ya.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:50] "GET /db/api/thread/details/?related=user&thread=1975 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:50] "GET /db/api/thread/details/?related=forum&thread=4541 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:24:57] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=89&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:04] "GET /db/api/forum/listUsers/?related=forum&limit=83&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:05] "GET /db/api/user/listPosts/?limit=28&user=ex4%40gmail.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:05] "GET /db/api/post/list/?post=109190&limit=71&order=asc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:05] "GET /db/api/forum/details/?related=user&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:05] "GET /db/api/forum/details/?related=user&forum=g91u HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:25:05] "GET /db/api/user/listFollowing/?limit=74&user=oxg4rh58%40list.ru&order=asc HTTP/1.1" 500 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:25:05] "GET /db/api/user/listFollowing/?limit=18&user=wz18xvh6%40ya.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:25:05] "GET /db/api/thread/listPosts/?limit=67&order=desc&thread=2165 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:05] "GET /db/api/thread/details/?related=user&related=forum&thread=9625 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:05] "GET /db/api/user/listFollowers/?limit=30&user=4sgpxngo%40bk.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:05] "GET /db/api/thread/details/?related=forum&thread=7740 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:13] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=73&order=desc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:13] "GET /db/api/forum/listPosts/?related=forum&related=user&limit=21&order=desc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:13] "GET /db/api/thread/listPosts/?limit=79&order=asc&thread=9303 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/forum/listPosts/?related=thread&related=user&related=forum&limit=51&order=asc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/thread/listPosts/?limit=72&order=desc&thread=8755 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/user/details/?user=d40f%40mail.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/user/listPosts/?limit=35&user=tvqo2%40yahoo.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/thread/listPosts/?limit=79&order=desc&thread=7665 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/post/details/?post=688227&related=thread&related=user&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/forum/details/?related=user&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/forum/details/?related=user&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/forum/details/?forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/thread/details/?related=forum&thread=2856 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:14] "GET /db/api/forum/details/?related=user&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:23] "GET /db/api/forum/listUsers/?limit=28&order=asc&forum=g91u HTTP/1.1" 200 -
Exception on /db/api/post/remove/ [POST]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/Post.py", line 129, in removePost
    if "post" in request.json:
TypeError: argument of type 'NoneType' is not iterable
10.20.0.53 - - [04/May/2016 15:25:23] "POST /db/api/post/remove/ HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:25:23] "GET /db/api/user/listFollowers/?limit=75&user=w5ge2%40mail.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:23] "GET /db/api/post/details/?post=812511 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:23] "GET /db/api/post/list/?post=494162&limit=43&order=asc&thread=7569 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:23] "GET /db/api/forum/details/?related=user&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:23] "GET /db/api/thread/listPosts/?limit=92&order=desc&thread=1730 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:23] "GET /db/api/user/listPosts/?limit=38&user=tcm0bg19%40gmail.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:23] "GET /db/api/user/listPosts/?limit=65&user=9vgec%40mail.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:23] "GET /db/api/thread/details/?related=user&thread=1382 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:24] "GET /db/api/forum/listThreads/?limit=77&order=desc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:24] "GET /db/api/thread/details/?related=user&thread=9787 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:24] "GET /db/api/user/listFollowers/?limit=84&user=hy%40list.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:24] "GET /db/api/thread/details/?thread=3108 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:33] "GET /db/api/forum/listUsers/?limit=57&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:33] "GET /db/api/forum/details/?forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:33] "GET /db/api/forum/listPosts/?related=user&related=forum&limit=29&order=asc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:35] "GET /db/api/thread/list/?forum=tomer&limit=95&order=desc&thread=1905 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:35] "GET /db/api/user/listFollowers/?limit=71&user=hxef82i%40ya.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:36] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=59&order=asc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=12&order=desc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/thread/listPosts/?limit=59&order=asc&thread=6620 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/user/details/?user=vbd4txc%40ua.ru HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/user/listFollowing/?limit=92&user=39kbl6%40mail.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/user/details/?user=opdw8bzu2g%40bk.ru HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/user/listFollowing/?limit=95&user=x9%40ya.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/thread/listPosts/?limit=91&order=asc&thread=5094 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/user/listPosts/?limit=45&user=c469shmfrg%40ua.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/user/listPosts/?limit=66&user=a5%40gmail.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=64&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:45] "GET /db/api/thread/details/?related=user&thread=9098 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:47] "GET /db/api/forum/listPosts/?related=user&related=thread&limit=66&order=desc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:47] "GET /db/api/forum/details/?related=user&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:47] "GET /db/api/user/listFollowers/?limit=14&user=8ilt5g%40mail.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:47] "GET /db/api/post/list/?post=587772&limit=58&order=desc&thread=7234 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:47] "GET /db/api/user/details/?user=dvil4%40mail.ru HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:25:47] "GET /db/api/user/listFollowing/?limit=47&user=bhqgz082p%40ya.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:25:47] "GET /db/api/forum/listPosts/?related=thread&limit=24&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:25:56] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=54&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:04] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=69&order=asc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:12] "GET /db/api/forum/listUsers/?related=forum&limit=36&order=asc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:12] "GET /db/api/post/list/?post=408533&limit=23&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:26:12] "GET /db/api/user/listFollowing/?limit=57&user=cgbtg39%40list.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:26:12] "GET /db/api/forum/details/?related=user&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:12] "GET /db/api/post/list/?post=694063&limit=22&order=desc&thread=2604 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:13] "GET /db/api/forum/listPosts/?related=forum&related=thread&related=user&limit=22&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:13] "GET /db/api/forum/listThreads/?related=forum&limit=48&order=asc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:13] "POST /db/api/user/create/ HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:14] "GET /db/api/forum/listPosts/?related=user&related=forum&limit=87&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:14] "GET /db/api/thread/listPosts/?limit=33&order=desc&thread=4587 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:22] "GET /db/api/forum/listUsers/?limit=95&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:22] "GET /db/api/user/listPosts/?limit=80&user=0lcmbvpn%40ya.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:22] "GET /db/api/post/details/?post=598630&related=thread&related=forum&related=user HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:23] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=70&order=desc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:23] "GET /db/api/forum/listThreads/?related=user&limit=23&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:23] "GET /db/api/user/listPosts/?limit=88&user=1o%40yahoo.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:23] "GET /db/api/thread/details/?related=user&thread=4427 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/forum/listThreads/?related=user&limit=66&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/user/listFollowers/?limit=75&user=dl9m%40gmail.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/thread/listPosts/?limit=52&order=asc&thread=5229 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/user/details/?user=go46r8gicv%40ya.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/user/details/?user=ln63tervo%40ya.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/user/details/?user=wgamkdx%40ua.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/post/details/?post=23087 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/post/list/?post=29736&limit=38&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/thread/listPosts/?limit=20&order=asc&thread=835 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/post/list/?post=931019&limit=19&order=asc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/post/details/?post=528177&related=thread&related=forum&related=user HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/forum/details/?related=user&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/post/details/?post=188845&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/post/details/?post=80986&related=thread&related=user&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/forum/details/?related=user&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:24] "GET /db/api/user/details/?user=hur%40ya.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:25] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=69&order=desc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:26] "GET /db/api/forum/listPosts/?related=thread&related=forum&related=user&limit=78&order=asc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:26] "GET /db/api/forum/listPosts/?related=user&limit=56&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:27] "GET /db/api/forum/listThreads/?limit=93&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:28] "GET /db/api/forum/listPosts/?related=forum&related=thread&limit=54&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:28] "GET /db/api/user/details/?user=bnrp3zmt1d%40ua.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:29] "GET /db/api/forum/listPosts/?related=thread&related=forum&related=user&limit=97&order=asc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:29] "GET /db/api/post/list/?post=530298&limit=51&order=desc&thread=7361 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:38] "GET /db/api/forum/listUsers/?limit=100&order=asc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:38] "GET /db/api/thread/listPosts/?limit=99&order=asc&thread=2497 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:26:38] "GET /db/api/user/listFollowing/?limit=71&user=sdpx9%40ua.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:26:38] "GET /db/api/user/listPosts/?limit=52&user=ybgplia7xc%40bk.ru&order=desc HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:26:38] "GET /db/api/user/listFollowing/?limit=40&user=z51kd3m%40bk.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:26:39] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=34&order=asc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:40] "GET /db/api/forum/listThreads/?related=user&limit=59&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:40] "GET /db/api/user/listFollowers/?limit=18&user=ngi%40bk.ru&order=desc HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:26:40] "GET /db/api/user/listFollowing/?limit=52&user=vba%40mail.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:26:40] "GET /db/api/thread/list/?forum=ql&limit=12&order=desc&thread=7855 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:40] "GET /db/api/user/listFollowers/?limit=35&user=rgua%40yahoo.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:41] "GET /db/api/forum/listPosts/?related=thread&limit=67&order=asc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:41] "GET /db/api/user/details/?user=qaexgmt0g1%40bk.ru HTTP/1.1" 200 -
Exception on /db/api/post/vote/ [POST]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/Post.py", line 192, in votePost
    if "vote" in request.json and "post" in request.json:
TypeError: argument of type 'NoneType' is not iterable
10.20.0.53 - - [04/May/2016 15:26:41] "POST /db/api/post/vote/ HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:26:41] "GET /db/api/user/listPosts/?limit=15&user=zi9g1up2es%40list.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:41] "GET /db/api/thread/listPosts/?limit=97&order=desc&thread=4563 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:42] "GET /db/api/forum/listThreads/?related=forum&limit=58&order=desc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:42] "GET /db/api/user/listFollowers/?limit=19&user=anegb%40ya.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:42] "GET /db/api/thread/list/?forum=se4iu&limit=25&order=asc&thread=3030 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:42] "GET /db/api/thread/details/?thread=9581 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:44] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=92&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:44] "GET /db/api/forum/details/?forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:44] "GET /db/api/forum/listPosts/?related=thread&limit=13&order=asc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:45] "GET /db/api/forum/listThreads/?related=user&limit=55&order=asc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:46] "GET /db/api/forum/listThreads/?related=user&limit=70&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:46] "GET /db/api/post/list/?post=837688&limit=33&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:46] "GET /db/api/thread/list/?forum=dltm4sp&limit=26&order=asc&thread=7211 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/forum/listUsers/?related=forum&limit=60&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/forum/listThreads/?limit=85&order=desc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/thread/listPosts/?limit=88&order=desc&thread=4566 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/user/listPosts/?limit=47&user=fg2bruw%40bk.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/user/details/?user=bmgr8%40yahoo.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/forum/details/?forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/user/listPosts/?limit=52&user=ki%40bk.ru&order=asc HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/user/listFollowing/?limit=74&user=edgk%40ua.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/thread/list/?forum=eazx1n2&limit=71&order=asc&thread=2873 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/forum/listThreads/?limit=85&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/forum/details/?forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/forum/listPosts/?limit=22&order=asc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:53] "GET /db/api/forum/listPosts/?related=thread&related=user&limit=12&order=asc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:54] "GET /db/api/post/list/?post=191948&limit=32&order=asc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:55] "GET /db/api/forum/listPosts/?related=thread&related=user&limit=98&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:26:55] "GET /db/api/forum/listPosts/?related=user&limit=38&order=desc&forum=tomer HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:26:55] "GET /db/api/user/listFollowing/?limit=94&user=f36%40bk.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:27:01] "GET /db/api/forum/listUsers/?related=forum&related=user&limit=49&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:01] "GET /db/api/post/list/?post=328711&limit=55&order=asc&thread=5796 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:01] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=62&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:01] "GET /db/api/user/listPosts/?limit=77&user=w1tqo9%40gmail.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:02] "GET /db/api/thread/list/?forum=cmq7igz2&limit=11&order=asc&thread=9794 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:02] "GET /db/api/post/details/?post=524769&related=forum&related=thread HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:02] "GET /db/api/thread/list/?limit=27&order=desc&thread=7930&user=avy3%40yahoo.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/forum/listUsers/?related=user&limit=99&order=asc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/thread/list/?limit=18&order=asc&thread=5526&user=ius%40list.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/thread/listPosts/?limit=98&order=asc&thread=3802 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/user/listFollowers/?limit=27&user=6iwsa%40list.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/forum/listPosts/?limit=39&order=desc&forum=cmq7igz2 HTTP/1.1" 200 -
Exception on /db/api/post/restore/ [POST]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/Post.py", line 148, in restorePost
    if "post" in request.json:
TypeError: argument of type 'NoneType' is not iterable
10.20.0.53 - - [04/May/2016 15:27:08] "POST /db/api/post/restore/ HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=75&order=desc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/thread/details/?thread=1043 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/forum/details/?related=user&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=49&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/post/list/?post=381489&limit=76&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/thread/listPosts/?limit=31&order=asc&thread=7087 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/thread/listPosts/?limit=75&order=asc&thread=2543 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/thread/listPosts/?limit=69&order=desc&thread=1914 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/thread/details/?related=forum&related=user&thread=1139 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/user/listPosts/?limit=30&user=rue8x30a9g%40yahoo.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:08] "GET /db/api/thread/listPosts/?limit=64&order=desc&thread=9452 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:14] "GET /db/api/forum/listUsers/?limit=49&order=asc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:14] "GET /db/api/user/listPosts/?limit=81&user=wfbs8mq5x%40gmail.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:15] "GET /db/api/forum/listPosts/?related=thread&limit=41&order=asc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:15] "GET /db/api/user/listFollowers/?limit=70&user=ug5ntx%40yahoo.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:20] "GET /db/api/forum/listUsers/?limit=57&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=56&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/post/details/?post=664840&related=thread&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/user/listFollowers/?limit=74&user=96z%40mail.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/forum/listPosts/?related=user&limit=66&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/post/details/?post=513323&related=thread&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/thread/details/?thread=665 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/thread/list/?forum=g91u&limit=32&order=desc&thread=3678 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/forum/listPosts/?limit=73&order=desc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/post/list/?post=579170&limit=80&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/thread/details/?related=user&thread=8101 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:21] "GET /db/api/forum/listPosts/?related=forum&limit=21&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:22] "GET /db/api/thread/list/?forum=cmq7igz2&limit=74&order=desc&thread=1678 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:22] "GET /db/api/post/list/?post=922763&limit=78&order=asc&thread=8869 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:22] "GET /db/api/forum/listPosts/?related=user&related=thread&limit=38&order=asc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:23] "GET /db/api/forum/listPosts/?related=forum&limit=80&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:23] "GET /db/api/thread/details/?related=forum&related=user&thread=1861 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:23] "GET /db/api/thread/list/?forum=cmxageot3&limit=21&order=desc&thread=6380 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:23] "GET /db/api/post/details/?post=173834&related=thread HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:23] "GET /db/api/post/details/?post=333628&related=forum&related=thread HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:27:23] "GET /db/api/user/listFollowing/?limit=81&user=r3218sg7%40ya.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:27:23] "GET /db/api/forum/details/?related=user&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:23] "GET /db/api/thread/details/?related=user&thread=2965 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:23] "GET /db/api/forum/listPosts/?limit=77&order=asc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:23] "GET /db/api/user/details/?user=cgv09%40gmail.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:28] "GET /db/api/forum/listUsers/?related=forum&related=user&limit=36&order=desc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:28] "POST /db/api/thread/close/ HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:30] "GET /db/api/forum/listThreads/?related=forum&limit=89&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:30] "GET /db/api/forum/details/?related=user&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:30] "GET /db/api/thread/list/?limit=80&order=desc&thread=4969&user=873l1kdn%40yahoo.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:30] "GET /db/api/thread/list/?forum=yfo2qgghp6&limit=35&order=asc&thread=3226 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:30] "GET /db/api/forum/listThreads/?related=user&limit=18&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:30] "GET /db/api/forum/listThreads/?limit=48&order=asc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:30] "GET /db/api/thread/list/?limit=53&order=desc&thread=1748&user=st0l%40ua.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:30] "GET /db/api/user/listPosts/?limit=83&user=zrd3%40mail.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:31] "GET /db/api/thread/details/?related=user&related=forum&thread=2036 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:37] "GET /db/api/forum/listUsers/?limit=47&order=asc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:37] "GET /db/api/post/list/?post=955681&limit=23&order=desc&thread=2594 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:27:37] "GET /db/api/user/listFollowing/?limit=85&user=zbfy38a%40mail.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:27:37] "GET /db/api/thread/list/?forum=cmq7igz2&limit=34&order=desc&thread=1676 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:37] "GET /db/api/post/list/?post=455812&limit=93&order=desc&thread=739 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:37] "GET /db/api/thread/list/?forum=g91u&limit=78&order=desc&thread=1888 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:43] "GET /db/api/forum/listUsers/?limit=40&order=desc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:43] "GET /db/api/thread/details/?thread=8096 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:43] "GET /db/api/thread/listPosts/?limit=41&order=asc&thread=227 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=52&order=desc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/user/listPosts/?limit=74&user=26lu1ir%40yahoo.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/forum/listThreads/?related=forum&limit=11&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/user/details/?user=bqialo0sm3%40ua.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/forum/listPosts/?limit=44&order=asc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/thread/list/?forum=tbrklfx&limit=75&order=asc&thread=4200 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/user/listPosts/?limit=79&user=g8ymbukpth%40yahoo.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/thread/listPosts/?limit=30&order=asc&thread=5339 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/forum/details/?related=user&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=22&order=desc&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/post/list/?post=731911&limit=55&order=desc&thread=4356 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/forum/details/?related=user&forum=tomer HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/user/listFollowing/?limit=99&user=1ruatwb2gk%40yahoo.com&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/thread/details/?thread=2963 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:44] "GET /db/api/user/listPosts/?limit=19&user=xur1awm%40yahoo.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/forum/listPosts/?related=thread&limit=100&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/user/details/?user=wf6m8dq%40list.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/post/list/?post=374839&limit=78&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/user/listFollowers/?limit=18&user=hpd%40gmail.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/user/listFollowers/?limit=38&user=qn%40yahoo.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/thread/list/?forum=cmq7igz2&limit=43&order=desc&thread=177 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/post/details/?post=201644&related=user HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/user/details/?user=8iodk1%40mail.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/forum/details/?related=user&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/post/details/?post=688060&related=thread HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/thread/details/?related=forum&related=user&thread=2325 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/post/list/?post=460546&limit=66&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:45] "GET /db/api/post/list/?post=610546&limit=97&order=desc&forum=tbrklfx HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:46] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=54&order=asc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:46] "GET /db/api/user/listPosts/?limit=79&user=i0k8pw%40bk.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:46] "GET /db/api/user/details/?user=cdt%40gmail.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:46] "GET /db/api/forum/details/?forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:46] "GET /db/api/forum/listThreads/?limit=23&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:46] "GET /db/api/thread/list/?limit=48&order=asc&thread=9997&user=0fk%40yahoo.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:46] "GET /db/api/user/listFollowers/?limit=95&user=zwub%40mail.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:46] "GET /db/api/user/details/?user=s8kw4mghq%40mail.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:47] "GET /db/api/forum/listThreads/?related=user&limit=61&order=asc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:52] "GET /db/api/forum/listUsers/?related=forum&related=user&limit=36&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:52] "GET /db/api/forum/details/?related=user&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:52] "GET /db/api/thread/listPosts/?limit=12&order=asc&thread=1151 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:52] "GET /db/api/post/list/?post=367676&limit=79&order=desc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:52] "GET /db/api/forum/details/?forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:52] "GET /db/api/post/list/?post=463052&limit=65&order=desc&thread=1451 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/thread/list/?forum=cmq7igz2&limit=13&order=desc&thread=3912 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/user/listFollowers/?limit=98&user=y09x7ocd%40gmail.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/thread/details/?related=user&related=forum&thread=10104 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/user/listFollowing/?limit=45&user=rxzwh3a%40bk.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/forum/listPosts/?related=forum&related=thread&related=user&limit=83&order=asc&forum=dltm4sp HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/forum/details/?related=user&forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/forum/details/?forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/post/details/?post=602401&related=user&related=thread HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/post/details/?post=94279 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/thread/details/?related=user&related=forum&thread=1796 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/user/details/?user=3wb864cu%40ya.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/thread/listPosts/?limit=48&order=asc&thread=4208 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:53] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=59&order=asc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:54] "GET /db/api/post/list/?post=961776&limit=87&order=asc&thread=6647 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:54] "GET /db/api/post/details/?post=372328&related=user&related=thread HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:54] "GET /db/api/user/details/?user=1g3s%40ua.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:54] "GET /db/api/user/details/?user=1k2my7bi%40yahoo.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:54] "GET /db/api/post/list/?post=915863&limit=99&order=asc&thread=472 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:54] "GET /db/api/thread/details/?related=user&thread=5643 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:54] "GET /db/api/user/details/?user=thyg01%40bk.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:27:54] "GET /db/api/thread/listPosts/?limit=33&order=desc&thread=6422 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:00] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=72&order=asc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:00] "GET /db/api/thread/listPosts/?limit=49&order=asc&thread=10058 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:00] "GET /db/api/forum/details/?related=user&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:00] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=39&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:01] "GET /db/api/forum/listPosts/?related=user&related=thread&related=forum&limit=38&order=asc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:01] "GET /db/api/thread/list/?limit=27&order=desc&thread=2543&user=h9%40bk.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:01] "GET /db/api/thread/details/?related=user&thread=3806 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:01] "GET /db/api/thread/details/?thread=3924 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:01] "GET /db/api/user/listFollowers/?limit=68&user=9g%40gmail.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:01] "GET /db/api/user/listFollowers/?limit=94&user=qbm0%40ua.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:01] "GET /db/api/thread/list/?forum=ql&limit=56&order=asc&thread=6972 HTTP/1.1" 200 -
Exception on /db/api/post/update/ [POST]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/Post.py", line 169, in updatePost
    if "post" in request.json and "message" in request.json:
TypeError: argument of type 'NoneType' is not iterable
10.20.0.53 - - [04/May/2016 15:28:01] "POST /db/api/post/update/ HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:28:01] "GET /db/api/user/listPosts/?limit=19&user=i1%40gmail.com&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:07] "GET /db/api/forum/listUsers/?related=forum&related=user&limit=97&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:07] "GET /db/api/thread/listPosts/?limit=44&order=desc&thread=817 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:07] "GET /db/api/user/details/?user=7np6%40yahoo.com HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:07] "GET /db/api/user/listFollowers/?limit=10&user=axvgtg%40bk.ru&order=desc HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:28:07] "GET /db/api/user/listFollowing/?limit=25&user=9k32c%40ya.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:28:07] "GET /db/api/forum/details/?forum=eazx1n2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:07] "GET /db/api/thread/details/?thread=1382 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:07] "GET /db/api/thread/list/?forum=cmq7igz2&limit=39&order=asc&thread=7650 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:13] "GET /db/api/forum/listUsers/?limit=84&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:13] "GET /db/api/post/details/?post=478064 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:19] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=79&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:19] "GET /db/api/forum/listPosts/?limit=66&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:19] "GET /db/api/user/listPosts/?limit=69&user=qi4xmg%40bk.ru&order=asc HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:28:19] "GET /db/api/user/listFollowing/?limit=34&user=nhkaf1s%40yahoo.com&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:28:19] "GET /db/api/thread/listPosts/?limit=93&order=asc&thread=335 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:19] "GET /db/api/post/details/?post=675080 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:19] "GET /db/api/user/listPosts/?limit=79&user=76pc4%40ua.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:20] "GET /db/api/forum/listThreads/?limit=96&order=desc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:20] "GET /db/api/forum/listPosts/?limit=69&order=desc&forum=yfo2qgghp6 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:20] "GET /db/api/forum/listPosts/?limit=59&order=asc&forum=ql HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:28:20] "GET /db/api/user/listFollowing/?limit=31&user=9ztepna%40ua.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:28:20] "GET /db/api/thread/list/?forum=cmq7igz2&limit=21&order=desc&thread=9273 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:26] "GET /db/api/forum/listUsers/?related=forum&related=user&limit=87&order=desc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:26] "GET /db/api/forum/listThreads/?limit=64&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:26] "GET /db/api/user/details/?user=zr0%40bk.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:32] "GET /db/api/forum/listUsers/?related=user&limit=97&order=desc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:32] "GET /db/api/forum/listThreads/?limit=36&order=asc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:32] "GET /db/api/post/details/?post=794101&related=user&related=thread&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:32] "GET /db/api/thread/details/?related=user&thread=9207 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:32] "GET /db/api/post/details/?post=565188&related=thread&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:32] "POST /db/api/thread/subscribe/ HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:33] "GET /db/api/forum/listPosts/?related=thread&related=forum&limit=72&order=asc&forum=g91u HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:33] "GET /db/api/user/listFollowers/?limit=75&user=ykdxrmfac%40ua.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:33] "GET /db/api/forum/listPosts/?related=forum&related=user&limit=80&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:33] "GET /db/api/thread/details/?related=user&related=forum&thread=6299 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:33] "GET /db/api/user/details/?user=vl3u%40list.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:33] "GET /db/api/post/list/?post=610178&limit=92&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:33] "GET /db/api/post/details/?post=261920&related=thread&related=user HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:33] "GET /db/api/forum/listPosts/?related=user&related=forum&limit=44&order=asc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:28:40] "GET /db/api/forum/listUsers/?limit=75&order=asc&forum=cmq7igz2 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:28:40] "GET /db/api/user/listFollowing/?limit=24&user=spv6y%40ua.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:28:40] "GET /db/api/thread/list/?forum=yfo2qgghp6&limit=19&order=asc&thread=2924 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:28:40] "GET /db/api/user/listFollowing/?limit=89&user=dz3%40gmail.com&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:29:07] "GET /db/api/thread/listPosts/?limit=74&order=desc&thread=7543 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:08] "GET /db/api/thread/list/?forum=se4iu&limit=82&order=asc&thread=2064 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:08] "GET /db/api/user/details/?user=cqdf%40ya.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:08] "GET /db/api/forum/listPosts/?related=user&limit=47&order=asc&forum=cmq7igz2 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:08] "GET /db/api/user/listFollowers/?limit=91&user=r46ul%40gmail.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:08] "GET /db/api/forum/listThreads/?related=user&limit=67&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:08] "GET /db/api/thread/listPosts/?limit=99&order=desc&thread=9486 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:08] "GET /db/api/user/details/?user=gg87p%40mail.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:08] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=47&order=asc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:08] "GET /db/api/forum/details/?related=user&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/thread/list/?forum=dltm4sp&limit=97&order=asc&thread=3927 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/post/details/?post=724131&related=forum HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/user/listPosts/?limit=99&user=r6k28w7t%40mail.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/user/listFollowers/?limit=34&user=vwmagxcgh%40gmail.com&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/forum/listThreads/?limit=80&order=desc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/user/listPosts/?limit=28&user=1p%40bk.ru&order=asc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/thread/details/?thread=6617 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/thread/details/?related=user&related=forum&thread=8248 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/user/details/?user=0gq4ib19%40ya.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/forum/details/?related=user&forum=tbrklfx HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:29:09] "GET /db/api/user/listFollowing/?limit=66&user=ug10%40bk.ru&order=asc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:29:16] "GET /db/api/forum/listUsers/?related=forum&limit=40&order=desc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:16] "GET /db/api/thread/listPosts/?limit=46&order=desc&thread=7485 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:16] "GET /db/api/user/listPosts/?limit=77&user=sgh9ba5x80%40ua.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:16] "GET /db/api/forum/listThreads/?related=user&related=forum&limit=45&order=desc&forum=ql HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:29:16] "GET /db/api/user/listFollowing/?limit=47&user=ldgau%40bk.ru&order=desc HTTP/1.1" 500 -
Exception on /db/api/thread/remove/ [POST]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/Thread.py", line 216, in removeThread
    if "thread" in request.json:
TypeError: argument of type 'NoneType' is not iterable
10.20.0.53 - - [04/May/2016 15:29:16] "POST /db/api/thread/remove/ HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:29:16] "GET /db/api/user/details/?user=8yg0%40ya.ru HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:16] "GET /db/api/post/list/?post=406634&limit=92&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:17] "GET /db/api/thread/list/?forum=tomer&limit=59&order=asc&thread=3867 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:17] "GET /db/api/thread/listPosts/?limit=20&order=asc&thread=7652 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:24] "GET /db/api/forum/listUsers/?related=user&related=forum&limit=67&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
Exception on /db/api/user/listFollowing/ [GET]
Traceback (most recent call last):
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 189, in listFollowing
    answer = getUsersInfoByEmail(emails)
  File "/home/tpadmin/TP_BD/bin/db_tp_kek/app/User.py", line 305, in getUsersInfoByEmail
    cursor.execute(sql)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/tpadmin/TP_BD/local/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")
10.20.0.53 - - [04/May/2016 15:29:24] "GET /db/api/user/listFollowing/?limit=83&user=s6q5xcr%40bk.ru&order=desc HTTP/1.1" 500 -
10.20.0.53 - - [04/May/2016 15:29:24] "GET /db/api/forum/listPosts/?related=user&limit=41&order=asc&forum=ql HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:24] "GET /db/api/forum/listThreads/?related=forum&related=user&limit=86&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:24] "GET /db/api/thread/list/?forum=cmxageot3&limit=15&order=asc&thread=2990 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:24] "GET /db/api/post/details/?post=466000&related=thread HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:30] "GET /db/api/forum/listUsers/?limit=96&order=desc&forum=cmxageot3 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:30] "GET /db/api/thread/listPosts/?limit=32&order=desc&thread=9408 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:30] "GET /db/api/user/listPosts/?limit=32&user=nc%40ya.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:31] "GET /db/api/post/list/?post=267102&limit=53&order=asc&thread=7785 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:31] "GET /db/api/thread/listPosts/?limit=89&order=desc&thread=896 HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:31] "GET /db/api/user/listPosts/?limit=27&user=fkvqtg8%40ya.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:31] "GET /db/api/forum/listPosts/?limit=78&order=asc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:37] "GET /db/api/forum/listUsers/?related=user&limit=43&order=desc&forum=se4iu HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:37] "GET /db/api/user/listPosts/?limit=50&user=q8%40ya.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:44] "GET /db/api/forum/listUsers/?limit=67&order=desc&forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:44] "GET /db/api/user/listPosts/?limit=52&user=6da1my0%40ya.ru&order=desc HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:44] "GET /db/api/forum/details/?forum=tomer HTTP/1.1" 200 -
10.20.0.53 - - [04/May/2016 15:29:44] "GET /db/api/user/listPosts/?limit=97&user=wk2%40mail.ru&order=asc HTTP/1.1" 200 -
